reference/5.1/Microsoft.PowerShell.Core/About/about_Using.md

```diff
@@ -1,46 +1,63 @@
 ---
 keywords: powershell,cmdlet
 locale: en-us
-ms.date: 08/15/2019
+ms.date: 01/29/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_using?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_Using
 ---
 # About Using
 
 ## SHORT DESCRIPTION
-Allows to indicate which namespaces are used in the session.
+Allows you to indicate which namespaces are used in the session.
 
 ## LONG DESCRIPTION
 
 The `using` statement allows you to specify which namespaces are used in the
 session. Adding namespaces simplifies usage of .NET classes and member and
-allows you to import classes from modules.
+allows you to import classes from script modules and assemblies.
 
 The `using` statement needs to be the first statement in the script.
 
-### Syntax
+The `using` statement should not be confused with the `using:` scope modifier
+for variables. For more information, see
+[about_Remote_Variables](about_Remote_Variables.md).
 
-To reference .NET Framework namespaces:
+## Syntax
+
+To preload typenames from a .NET namespace:
 
 ```
-using namespace <.NET-framework-namespace>
+using namespace <.NET-namespace>
 ```
 
-To reference PowerShell modules:
+To load classes from a PowerShell module:
 
 ```
 using module <module-name>
 ```
 
-> [!NOTE]
-> `Import-Module` and the `#requires` statement only import the module
-> functions, aliases, and variables, as defined by the module. Classes are not
-> imported. The `using module` statement imports the classes defined in the
-> module. If the module isn't loaded in the current session, the `using`
-> statement fails.
+To preload classes from a .NET assembly:
+
+```
+using assembly <.NET-assembly-path>
+using assembly <.NET-namespace>
+```
+
+Loading a namespace makes it easier to reference types by their short names.
```

Comment:
This isn't really a form of loading. Just adds places to look for types.

Reply:
Fixed